### PR TITLE
New Terraform template - EventBridge Scheduler to CloudWatch Alarm

### PR DESCRIPTION
**Pull request by edwin-so**
*Issue #2172, if available:*

*Description of changes:*
New Terraform template that creates two EventBridge Scheduler schedules to enable and disable the action of a CloudWatch alarm

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# AWS Service 1 to AWS Service 2
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:1]: Please be more descriptive

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+This pattern will create two EventBridge Scheduler schedules to enable and disable the action of a CloudWatch alarm in a specific period. This example will enable the alarm at 08:00 and disable it at 17:00 in the US/Eastern timezone.
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:3]:
```suggestion
This pattern will create two Amazon EventBridge Scheduler schedules to enable and disable the action of a Amazon CloudWatch alarm in a specific period. This example will enable the alarm at 08:00 and disable it at 17:00 in the US/Eastern timezone.
```

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    ```
+1. From the commend line, apply the configuration in the main.tf file and follow the prompts: 
+    ```
+    terraform apply
+    ```
+
+
+## How it works
+
+Two Amazon EventBridge Scheduler schedules will be used to enable and disable the action of a CloudWatch alarm. The Terraform template creates a VPC, an EC2 instance and two schedules that invokes the enableAlarmActions and disableAlarmActions API regularly.
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:38]: Why is the creation of a VPC and EC2 instance necessary?

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Cleanup
+ 
+1. Delete all created resources and follow prompts:
+    ```
+    terraform destroy
+    ```
+
+----
+Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:54]:
```suggestion
Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/example-pattern.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
+{
+  "title": "EventBridge Scheduler to CloudWatch Alarm",
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/example-pattern.json:2]:
```suggestion
  "title": "EventBridge Scheduler for CloudWatch Alarm",
```

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/example-pattern.json [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    "text": [
+      "Delete the stack: <code>terraform destroy</code>."
+    ]
+  },
+  "authors": [
+    {
+      "name": "Edwin So",
+      "image": "https://media.licdn.com/dms/image/C5603AQGXFWV-L0nYuw/profile-displayphoto-shrink_200_200/0/1596368326454?e=1715212800&v=beta&t=-Xc2wRWeM1TiNpldpkHC7RsgjjzJJ17U0np-VQs0ij0",
+      "bio": "Cloud Support Engineer @ AWS | CloudWatch SME",
+      "linkedin": "https://www.linkedin.com/in/edwin-so/"
```
Comment (bfreiberg) [inline eventbridge-schedule-to-cloudwatch-alarm-terraform/example-pattern.json:50]:
```suggestion
      "linkedin": "edwin-so"
```

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md (hunk 3 above)
Comment (edwin-so) [reply eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:38]: Hi Ben, thanks for checking. The VPC and EC2 creation are for producing the CPUUtilization metric. Actually I just need a sample metric for the CloudWatch alarm. And I think CPUUtilization would be something that our customer usually use.

File: eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md (hunk 1 above)
Comment (edwin-so) [reply eventbridge-schedule-to-cloudwatch-alarm-terraform/README.md:1]: Sorry, I forgot to update this line. Should I use "EventBridge Scheduler for CloudWatch Alarm" for this?

**Comment by bfreiberg** [comment]
Looks good, thank you for your contribution. A Developer Advocate will merge the changes and publish the pattern to [serverlessland.com](http://serverlessland.com/).

**Pull request merged**